src/probes/isRequire.js

```diff
@@ -99,7 +119,7 @@ function main(node, options) {
       analysis.addWarning("unsafe-import", null, node.loc);
 
       // We skip walking the tree to avoid anymore warnings...
-      return ProbeSignals.Skip;
+      return Symbol.for("skipWalk");
```

Comment:
Need to keep the code with ProbeSignals here

Reply:
thanks its fixed in latest commit